index.html

```diff
@@ -94,17 +94,17 @@ <h1 class="project-title">Tonic</h1>
     <div class="projects-card">
       <div class="card-header">
         <div class="projects">
-          <img src="./Images/Snapshoot Portfolio__1.png" alt="" class="project-screenshot">
+          <img src="./Images/Snapshoot-Portfolio__1.png" alt="Depiction of Project two" class="project-screenshot">
```

Comment:
Kindly check your image path, your images are not displaying 😊

![image](https://user-images.githubusercontent.com/73854999/200681817-20156910-d7fa-4e33-9804-80259c76b36d.png)